py/compile.c

```diff
@@ -782,6 +782,32 @@ STATIC qstr compile_funcdef_helper(compiler_t *comp, mp_parse_node_struct_t *pns
     // compile the function definition
     compile_funcdef_lambdef(comp, fscope, pns->nodes[1], PN_typedargslist);
 
+    #if MICROPY_ENABLE_DOC_STRING
```

Comment:
Can we update the comment in `mpconfig.h` to explain that this enables docstrings on functions, modules, and classes?

(I think we should keep them as one combined option)